src/ducks/recurrence/RecurrencePage/RecurrencePage.jsx

```diff
@@ -7,7 +7,7 @@ import {
   hasQueryBeenLoaded
 } from 'cozy-client'
 import { recurrenceConn, RECURRENCE_DOCTYPE } from 'doctypes'
-import { bundleTransactionsQueryConn } from './queries'
+import { bundleTransactionsQueryConn } from '../queries'
```

Comment:
nit: Favor use of aliases 'ducks/recurrence/queries'

Reply:
oui c'est fait plus loin, ce commit ne sert qu'à faire le split, sans changement